yal/assessment_data/body_image.py

```diff
@@ -0,0 +1,53 @@
+# Assessment state naming convention
+# state_a[number]_[subsection]_q[question]_topic
+ASSESSMENT_QUESTIONS = {
+    "8":
+    # A6 Body Image
+    {
+        "start": "state_a6_start",
+        "questions": {
+            "state_a6_start": {
+                "text": "\n".join(
+                    [
+                        "[persona_emoji]  *Let me know how you feel about these next "
+                        "few statements.*",
+                        "",
+                        "There are no right or wrong answers. Try and answer "
+                        "honestly and freely as you can.",
+                        "",
+                        "Remember — everything stays between us! 🤐",
+                        "",
+                    ]
+                ),
+                "options": ["OK, let's do it!"],
+                "next": "state_a6_q1_body_image",
+            },
+            "state_a6_q1_body_image": {
+                "text": "\n".join(
+                    [
+                        "*_I feel good about myself._*",
+                    ]
+                ),
+                "options": [
+                    "Strongly Agree",
+                    "Kinda Agree",
+                    "Do not Agree",
+                ],
+                "next": "state_a6_q2_body_image",
+            },
+            "state_a6_q2_body_image": {
+                "text": "\n".join(
+                    [
+                        "*_I feel good about my body._*",
+                    ]
+                ),
+                "options": [
+                    "Strongly Agree",
+                    "Kinda Agree",
+                    "Do not Agree",
+                ],
+                "next": None,
```

Comment:
We probably need `type`s for these questions, otherwise they get treated like `info`, so they'll only show the text and then not wait for user input